parser/types/field_type.go

```diff
@@ -53,7 +53,8 @@ type FieldType struct {
 	// collate represent collate rules of the charset
 	collate string
 	// elems is the element list for enum and set type.
-	elems []string
+	elems            []string
+	elemsIsBinaryLit []bool
```

Comment:
I should have documented that `jsonFieldType` is ought to be modified together with `FieldType` to make (de)serialization work after that refactor PR.

Maybe you can do that in this PR?

Reply:
OK, thanks for pointing out.